python3: can't open file '/home/pkc/ops445/lab5/./CheckLab1.py': [Errno 2] No such file or directory
